core/arch/arm/crypto/aes-gcm-ce.c

```diff
@@ -1,8 +1,6 @@
+// SPDX-License-Identifier: BSD-2-Clause
 /*
  * Copyright (c) 2017, Linaro Limited
- * All rights reserved.
- *
- * SPDX-License-Identifier: BSD-2-Clause
  */
```

Comment:
This change generates several of this kind:
```c
/* SPDX-License-Identifier: BSD-2-Clause */
/*
 * Copyright (c) 2017, Linaro Limited
 */
```
I find it a bit ugly.

Post process all such headers (with any dating) into something like this?
```c
/* SPDX-License-Identifier: BSD-2-Clause */
/* Copyright (c) 2017, Linaro Limited */
```

Reply:
Maybe, maybe not. In fact I think three line makes it a bit easier to read ;-)
IMO it is not specific to copyright/license headers, and more related to the way any comment may or may not be formatted. We don't disallow such 3-line comments, generally speaking, so I'd rather not introduce more recommendations/rules.

Reply:
or
// SPDX-License-Identifier: BSD-2-Clause
// Copyright (c) 2017, Linaro Limited

which look nice as a short source notice. Thanks @jforissier for link.